system-test/.eslintrc.yml

```diff
@@ -2,4 +2,4 @@
 env:
   mocha: true
 rules:
-  no-console: off
\ No newline at end of file
+  no-console: off
```

Comment:
Could we drop these two lines, and just delete the console logs from the system test?

Reply:
Yes! https://github.com/googleapis/gapic-generator-typescript/pull/153